docs/dev/guidelines/client-tests.rst

```diff
@@ -314,3 +311,20 @@ Some guidelines:
   +--------------------------------------------------------+-----------------------------------------------------------------+
   | A spy should have been called once                     | :code:`expect(spy).toHaveBeenCalledTimes(1);`                   |
   +--------------------------------------------------------+-----------------------------------------------------------------+
+  | A spy should have been called with a value             | Always test the number of calls as well:                        |
+  |                                                        |                                                                 |
+  |                                                        | .. code:: ts                                                    |
+  |                                                        |                                                                 |
+  |                                                        |     expect(spy).toHaveBeenCalledTimes(1);                       |
+  |                                                        |     expect(spy).toHaveBeenCalledWith(value);                    |
+  |                                                        |                                                                 |
+  |                                                        | If you have multiple calls, you can verify the parameters       |
+  |                                                        | of each call separately:                                        |
+  |                                                        |                                                                 |
+  |                                                        | .. code:: ts                                                    |
+  |                                                        |                                                                 |
+  |                                                        |     expect(spy).toHaveBeenCalledTimes(3);                       |
+  |                                                        |     expect(spy).toHaveBeenNthCalledWith(0, value0);             |
+  |                                                        |     expect(spy).toHaveBeenNthCalledWith(1, value1);             |
+  |                                                        |     expect(spy).toHaveBeenNthCalledWith(2, value2);             |
```

Comment:
```suggestion
  |                                                        |     expect(spy).toHaveBeenNthCalledWith(1, value0);             |
  |                                                        |     expect(spy).toHaveBeenNthCalledWith(2, value1);             |
  |                                                        |     expect(spy).toHaveBeenNthCalledWith(3, value2);             |
```
https://jestjs.io/docs/expect#tohavebeennthcalledwithnthcall-arg1-arg2-

Reply:
These guys can't be serious o_O